### PR TITLE
Change Zstd compression default to 19

### PR DESCRIPTION
**Pull request by jakirkham**
### Description

This uses less memory and takes less time while still providing a good level compression. Also lines up with how this was changed in conda-package-handling and conda-forge.

xref: https://github.com/conda/conda-package-handling/pull/169
xref: https://github.com/conda/conda-package-handling/issues/168
xref: https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/217

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @mbargull @dholth @jezdez

**Comment by mbargull** [comment]
Could you also change/remove ` Defaults to the maximum.` from the help text in `conda_build.cli.main_build`?

File: conda_build/cli/main_build.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def parse_args(args):
     p.add_argument(
         "--zstd-compression-level",
         help=("When building v2 packages, set the compression level used by "
-              "conda-package-handling. Defaults to the maximum."),
+              "conda-package-handling. Defaults to 19."),
```
[the remark this answers is not in the document]
Comment (jakirkham) [reply conda_build/cli/main_build.py]: Actually we can reuse the variable here. Updating...

**Review by mbargull — APPROVED** [review]
LGTM, thanks!

**Review by jezdez — APPROVED** [review]

**Review by dholth — APPROVED** [review]
Lgtm

**Comment by mbargull** [comment]
One, two, three, merge :).

**Comment by jakirkham** [comment]
Thanks all! 🙏

**Pull request merged**